test/Mono.Linker.Tests.Cases/Interop/PInvoke/Warnings/ComPInvokeWarning.cs

```diff
@@ -175,7 +175,7 @@ static void Call_CanSuppressPInvokeWithRequiresUnreferencedCode ()
 		static extern void CanSuppressPInvokeWithRequiresUnreferencedCode (IFoo foo);
 
 		[ExpectedWarning ("IL2050")]
-		[ExpectedWarning ("IL2026")]
+		[ExpectedWarning ("IL2026", ProducedBy = ProducedBy.Trimmer | ProducedBy.RequiresUnreferencedCodeAnalyzer)]
 		static void Call_PInvokeWithRequiresUnreferencedCode ()
```

Comment:
Should we have this test in another file that tests using all analyzers? since is an interaction between them

Reply:
I think we want to move there little by little (have all/most tests in Mono.Linker.Tests be run using all analyzers or a combination of them via the `ProducedBy` property). We can run tests under `Interop` with the `Requires_` analyzers, but I'd prefer doing that in another PR.